src/training/graph_group_singleton.cpp

```diff
@@ -18,12 +18,7 @@ void SingletonGraph::execute(Ptr<data::Batch> batch) {
 
   // Get batch stats
   size_t batch_words = batch->wordsTrg();
-
-  if(scaleLearningRate_) {
-    opt_->update(graph_, batch_words / avgBatchWords_);
-  } else {
-    opt_->update(graph_);
-  }
+  opt_->update(graph_);
```

Comment:
How come I don't see this change for graph_group_sync.cpp? I like that, since I am modifying that file :)